### DISCUSSION TITLE
Cerrar sesión as button link or row?

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1682] [Proposals]
### Goal

¿Qué solución es más correcta y ejecutable en Mistica?
<img width="885" alt="image" src="https://github.com/Telefonica/mistica-design/assets/95479574/5a5e8b6d-4583-414e-a3b1-cf5bd5e3a2d5">

### Context

Nuestro objetivo es poder tener una acción destructiva en la parte final de la página, en Moves quieren un buttonLink desctructuvo centrado, para O2ES se plantea como row..

### Impact

CMS needed

### Discussion Questions

-¿Puede una row ser destructiva? ¿Existe esa prop¿
-Se puede alinear a la izquierda un botón si está en un button layout?

### Do you have a visual proposal?

https://www.figma.com/file/eE87dEhhkHw8esqhfvoZ0c/Figma-para-poner-ejemplos-de-cosas?type=design&node-id=270-3735&mode=design

**Comment by aweell** [comment]
Buenas @MontseTelefonica,


> -¿Puede una row ser destructiva? ¿Existe esa prop¿

No, no existe esa prop

> -Se puede alinear a la izquierda un botón si está en un button layout?

Entiendo que esta pregunta está relacionada con:
https://github.com/Telefonica/mistica-design/discussions/1683

Aquí lo ideal es que las dos experiencias sean iguales. En el hipotético caso de que fuera posible esa row con acción destructiva, evitaría la lógica de que en Moves sea un botón y en O2 sea una row.

A día de hoy es posible centrar ese button link como propone Moves: [Ejemplo](https://tinyurl.com/25hoo75u)

**Comment by MontseTelefonica** [comment]
Hola @aweell 
Nos hemos percatado que en en Vivo usan row roja. ¿Es un mal uso de Mistica?
![Image (4)](https://github.com/Telefonica/mistica-design/assets/95479574/aac1613e-c163-4ece-bea9-0bf3bf6b2ea4)

**Comment by aweell** [comment]
Te planteo los diversos dilemas que aparecen al plantear esta variante de las rows:

* Actualmente los button link danger no llevan chevron a diferencia de la variante default, en el caso de las listas, debería llevar chevron o no un item destructivo.
* En el caso de que se decidiera que no llevan chevron porque se considera que siempre son una acción, pueden otros items de la lista que no sean destructivos ser una acción, no llevar chevron? Debería ser configurable que item lo lleva o cual no?
* En todos los casos se pueden agrupar esas acciones destructivas dentro de la categoría de elementos que recoge la lista?
* Que pasa con los otros elementos de la lista (subtitle, description?)
* Que pasa en el contexto inverse?
* En los casos en los que una navegación lleve a un segundo paso que es destructivo, debería pintarse como destructivo el item en la lista? [Ejemplo](https://www.figma.com/file/cqDlZbbqsoCwNOZgzoT33K/Settings-TEST?type=design&node-id=1807%3A14191&mode=design&t=ni7ufWtSkPWlM7QW-1)

**Comment by yceballost** [comment]
#1708
